infl-components-src/tooltip/index.js

```diff
@@ -59,7 +59,7 @@ export default class Tooltip extends Component {
   static propTypes = {
     title:             PropTypes.string,
     element:           PropTypes.node.isRequired,
-    position:          PropTypes.oneOf(['top', 'bottom']),
+    position:          PropTypes.string,
```

Comment:
i would just remove it then, unless we want to keep it for self-documentation, in which case is should be [top, bottom, left, right]